### DISCUSSION TITLE
How to embed monitoring tool codes in the app

### DISCUSSION THREAD

**Discussion by diegoleme** [directus/directus#15816] [Feature Requests]
I would like to include third-party scripts to monitor the Directus app in the browser. (Ex.: Sentry, New Relic, ELK...)

I didn't find a way to do this. Would it make sense to have a specific extension for this?

**Comment by azrikahar** [comment]
It's currently not possible to include third-party script in the app, but there's an existing discussion #12266 for custom scripts so feel free to give that an upvote!

Just in case anyone else lands in this discussion in regards to third-party monitoring integration in general but not specific to the app, there's another discussion #12626 that details how to include it for the API/server side.

**Reply by diegoleme** [reply]
Thanks @azrikahar. I'm going to the other thread

**Comment by nickrum** [comment]
This can be achieved through the `embed` hook 👍
